micro_brewery/data/pos_config.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="point_of_sale.pos_config_main" model="pos.config">
+        <field name="start_category" eval="1"/>
+        <field name="limit_categories" eval="1"/>
+        <field name="iface_available_categ_ids" eval="[(6, 0, [ref('pos_category_3')])]"/>
+        <field name="iface_start_categ_id" ref="pos_category_3"/>
+    </record>
```

Comment:
Ok so here is the big remark: this is a bad practice (sorry) to update standard data, since it could already be in use when the user installs the industry. So from now on, we will only create new records of `pos.config`.
So don't forget to add the what was removed previously: `name`, `down_payment_product_id`, 'active', ...